app/models/concerns/queue_it/queable.rb

```diff
@@ -37,20 +54,13 @@ def get_next_node_in_queue
 
     def formatted_queue(nodable_attribute)
       return if local_queue.empty?
+      return [local_queue.head_node.nodable.send(nodable_attribute)] if local_queue.one_node?
 
-      if local_queue.one_node?
-        [local_queue.head_node.nodable.send(nodable_attribute)]
-      elsif local_queue.two_nodes?
+      if local_queue.two_nodes?
         [local_queue.head_node.nodable.send(nodable_attribute),
-          local_queue.tail_node.nodable.send(nodable_attribute)]
+         local_queue.tail_node.nodable.send(nodable_attribute)]
```

Comment:
Why not use `formatted_generic_queue` here?

Reply:
Code legacy. I'll updated.